### PR TITLE
pterodactyl installation and setup tutorial

### PR DESCRIPTION
**Pull request by Marcel2508**
This adds a tutorial for installing and initial configuration for pterodactyl game server panel.
This includes only bare minimum setup and could be extended further later on.

I have read and understood the Contributor's Certificate of Origin at the end of the [template](https://github.com/netcup-community/community-tutorials/blob/main/TEMPLATE.md) and I hereby certify that I meet the contribution criteria described in it. 

Signed-off-by: Marcel Aust <nc-wiki@marcel-aust.de>

### CODE REVIEW

File: community-tutorials/pterodactyl-panel-install-ubuntu/01-en.md [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+![Create new location create](create-new-location-data.jpg)
+
+
+To configure your new node navigate to the **Nodes** tab and click **Create New**.
+
+![create new node](create-new-node.jpg)
+
+Here you need to fill in all the required information. Tailor the values to your need. It's important to configure the domain the same as used for your letsencrypt configuration!
+
+![configure new node](configure-new-node.jpg)
```
Comment (raphi-0901) [inline community-tutorials/pterodactyl-panel-install-ubuntu/01-en.md:226]: I had to fill all fields.

**Review by raphi-0901 — CHANGES_REQUESTED** [review]

**Comment by raphi-0901** [comment]
I set up the Minecraft-Server and tried to connect to it. It then gives me following error: `io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information`

**Comment by Marcel2508** [comment]
> I set up the Minecraft-Server and tried to connect to it. It then gives me following error: `io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information`

For me the setup worked. I was able to connect to my server successfully. 
I could add a notice within the requirements-section that port 25565 should be open (ufw, iptables etc). But I don't think networking / configuring docker / the server should be within the scope of this tutorial or?

**Comment by raphi-0901** [comment]
This was my fault. I forgot to open port 25565. Could you please add a hint? You don´t have to go into detail with this.

File: community-tutorials/pterodactyl-panel-install-ubuntu/01-en.md (hunk 1 above)
Comment (Marcel2508) [reply community-tutorials/pterodactyl-panel-install-ubuntu/01-en.md:228]: Modified the statement to "Here you need to fill in all fields".

**Comment by Marcel2508** [comment]
> This was my fault. I forgot to open port 25565. Could you please add a hint? You don´t have to go into detail with this.

Added a notice to requirements section.

**Pull request merged**